ch04packaging/04documentation.ipynb

```diff
@@ -134,15 +134,22 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Set up sphinx"
+    "### Set up Sphinx"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install Sphinx using the [appropiate instructions](https://www.sphinx-doc.org/en/master/usage/installation.html) for your system following the documentation online."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "\n",
-    "Invoke the [sphinx-quickstart](https://www.sphinx-doc.org/en/latest/usage/quickstart.html) command to build Sphinx's\n",
+    "Invoke the [sphinx-quickstart](https://www.sphinx-doc.org/en/master/usage/quickstart.html) command to build Sphinx's\n",
```

Comment:
Why have they got rid of `latest`? 😠  The problem if we refer to `master` is that we are pointing to the unreleased version of sphinx... and the documentation may not match with the version they have installed. Hopefully they won't change much the quickstart tool.

Reply:
It is a little annoying. We could point it at the `3.x` version instead as that's likely the version they'll have if they installed it recently?

Reply:
We can add a comment that the output they see from the quickstart tool may vary a bit from what's in the notes or the instructions we point them to.

Reply:
Sounds good - I'll add that and then merge.